tools/rna_tools/exparna/exparna.xml

```diff
@@ -1,8 +1,8 @@
-<tool id="exparna" name="ExpaRNA" version="0.1.0">
+<tool id="exparna" name="ExpaRNA" version="1.0.1">
     <description></description>
     <requirements>
-        <requirement type="package" version="1.0">exparna</requirement>
-        <requirement type="package" version="1.8.5">vienna_rna</requirement>
+        <requirement type="package" version="1.0.1">exparna</requirement>
```

Comment:
https://github.com/bioconda/bioconda-recipes/blob/master/recipes/exparna/meta.yaml
exparna has a dependency defined on viennarna, so this can be dropped.